src/main/java/transfer/countrytrans/CountryApi.java

```diff
@@ -0,0 +1,13 @@
+package transfer.countrytrans;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import transfer.countrytrans.routers.Country;
+
+@Getter
+@EqualsAndHashCode
+@ToString
+public class CountryApi {
```

Comment:
Should be 1 class Api with country and state, not 2 classes StateApi, CountryApi

Reply:
changed